### DISCUSSION TITLE
slf4j interprets java-agent log.info as [ERROR]

### DISCUSSION THREAD

**Discussion by kamal-kaur04** [qos-ch/slf4j#284] [Q&A]
I have a java-agent in which I'm using slf4j for logging. But, terminal shows "[ERROR]" log as well in the `mvn test` output. Any help around this would be appreciated.

Using `<slf4j.version>2.0.0-alpha7</slf4j.version>`

Thanks.

<img width="166" alt="Screenshot 2022-04-26 at 7 02 43 PM" src="https://user-images.githubusercontent.com/38219887/165311576-19491cc4-05f1-4577-95a4-c6a20d32b7ad.png">

**Comment by ExE-Boss** [comment]
This is most likely because `slf4j‑simple` uses `System.err` instead of `System.out`, so that applications which transform text as part of a shell pipeline can easily disambiguate the logging information from actual output.
